Ai Pin/Views/Settings/AccentColorView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  AccentColorView.swift
+//  Ai Pin
+//
+//  Created by Diego Gutierrez on 27/04/24.
+//
+
+import SwiftUI
+
+struct AccentColorView: View {
+    @EnvironmentObject private var colorStore: ColorStore
+    
+    @State private var accentColor = Color.blue
+    @State private var originalColor = Color.blue
+    @State private var triggerSave = false
+    
+    @Environment(\.dismiss) private var dismiss
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("This is an example title")
+                .foregroundStyle(accentColor)
+                .padding([.bottom], 5)
+                .bold()
+                .font(Font.system(size: 20))
+            Text("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum sem lacus, scelerisque ut ultrices in, aliquam ut massa. Vestibulum sed iaculis diam. In pulvinar felis nibh, ultrices bibendum tellus elementum ut. Ut non turpis feugiat, molestie justo sed.")
+                .foregroundStyle(.foreground)
+        }
+        .padding()
+        .background(.thinMaterial)
+        .clipShape(RoundedRectangle(cornerRadius: 25.0))
+        .padding(13)
+        .padding([.top], 20)
+        .onAppear() {
+            let tempColor = colorStore.loadColor()
+            accentColor = tempColor
+            originalColor = tempColor
+        }
```

Comment:
if this is a "preview" it is better to use the actual view being previewed with fake data instead.